tests/distributed/test_chunked_prefill_distributed.py

```diff
@@ -52,6 +54,7 @@ def test_models(
             model,
             dtype=dtype,
             tensor_parallel_size=2,
+            enforce_eager=bool(is_hip()),
```

Comment:
nit: Why do we need `bool`?

Reply:
Thanks. Removed bool. Was misled by ruff's suggestion.